### PR TITLE
GITPB-245 Apply for an event wizard

### PR DESCRIPTION
**Pull request by jebw**
### JIRA ticket number

GITPB-245

### Context

The Apply an Event wizard should capture and validate the steps for the Apply for an Event process

### Changes proposed in this pull request

1. Integrated GovUK Form Builder
2. Added email address validation to Personal Details step
3. Added Contact Details step
4. Added Further Details step

### CODE REVIEW

File: app/helpers/events_helper.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module EventsHelper
+  def format_event_date(event)
+    return if event.startDate.blank?
```
Comment (ethax-ross) [inline app/helpers/events_helper.rb:3]: The camel case here makes me cringe - do you want me to update the API to spit out snake case? I think it's just a serialiser setting

File: spec/controllers/event_steps_controller_spec.rb [hunk 2]
```diff
@@ -1,8 +1,18 @@
 require "rails_helper"
```
Comment (ethax-ross) [inline spec/controllers/event_steps_controller_spec.rb:1]: I don't feel strongly about it, but I would expect request specs to live in `/spec/requests/`

Comment (jebw) [reply spec/controllers/event_steps_controller_spec.rb:1]: Yeh, good call - I'll move all of them.

File: app/helpers/events_helper.rb (hunk 1 above)
Comment (jebw) [reply app/helpers/events_helper.rb:3]: Agreed, it was done to maintain parity with the API. I'm in two minds whether the appropriate option is to amend the case here or in the API. It is orthoganol to these changes though so I'd suggest doing that in a separate PR?

Comment (ethax-ross) [reply app/helpers/events_helper.rb:3]: I would vote for changing the API - it's more than likely going to be consumed by Rails apps predominantly so I think it makes sense to change it there; I'll take a look into it

File: spec/controllers/event_steps_controller_spec.rb (hunk 2 above)
Comment (jebw) [reply spec/controllers/event_steps_controller_spec.rb]: Ok moved.

**Pull request merged**